.github/workflows/check-lib.yml

```diff
@@ -204,22 +204,22 @@ jobs:
           - lavalink
         features:
           - native
-          - rustls
+          - rustls-native-roots
```

Comment:
Should there be a test for `rustls-webpki-roots` too or in a followup?

Reply:
@vilgotf wanted to rework CI to use powersets, so it will be done at some later point in time

Reply:
K